examples/Clock/ClockDS.py

```diff
@@ -28,6 +36,11 @@ def time(self):
     def read_gmtime(self):
         return time.gmtime()
 
+    @attribute(dtype='DevEnum', enum_labels=get_enum_labels(Noon))
```

Comment:
Thanks a lot for the PR, it looks great ! :)

What about supporting python enum as `enum_labels` argument? This would simplify the example even more.

Reply:
Interesting that you suggest that - I actually started out with it that way, but then changed my mind.  Setting `enum_labels` equal to something that isn't a sequence of labels didn't feel quite right.    Do you still want that option?

I had it here (I think):
https://github.com/tango-controls/pytango/blob/01f92b9/tango/attr_data.py#L170
Something like:
```
if isbaseclass(v, enum.Enum):
    v = get_enum_labels(v)
```
Not sure if that would be the right place to put it?

Reply:
Yes, that sounds like a nice option.  Then you we don't include the `enum_labels` in the decorator, so just the very simple: `@attribute(dtype=Noon)`

Was going to add it, but I see you already merged!  I'll try it out, and see if I can make another PR some time this week.

Reply:
@ajoubertza I just created an issue about it: #195